### ISSUE TITLE
[Android/Wasm] Accessing control with DeferLoadStrategy=Lazy from code-behind cause NRE

### ISSUE THREAD

**Issue by XiaotianNetlift** [unoplatform/Uno.WindowsCommunityToolkit#50]
## I'm submitting a...
- Bug report (I searched for similar issues and did not find one)

## Current behavior
Clicking on navigation bar item cause NullRerefenceException to be thrown.

## Expected behavior
Clicking on navigation bar item should open the sample picker grid.

## Minimal reproduction of the problem with instructions
Click on any of navigation bar (`NavigationView`) items ("Controls", "Helpers", "Extensions")
Nothing is else happening, beside this error in the log:
```
{System.NullReferenceException: Object reference not set to an instance of an object.
  at Microsoft.Toolkit.Uwp.SampleApp.Shell.NavView_ItemInvoked (Windows.UI.Xaml.Controls.NavigationView sender, Windows.UI.Xaml.Controls.NavigationViewItemInvokedEventArgs args) [0x00016] in D:\code\framework\Uno.WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp.Shared\Shell.SamplePicker.cs:132 
  at Microsoft.Toolkit.Uwp.SampleApp.Shell+<>c__DisplayClass39_3.<InitializeComponent>b__18 (Windows.UI.Xaml.Controls.NavigationView NavView_ItemInvoked_sender, Windows.UI.Xaml.Controls.NavigationViewItemInvokedEventArgs NavView_ItemInvoked_args) [0x00000] in D:\code\framework\Uno.WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp.Droid\obj\Debug\80\g\XamlCodeGenerator\Shell_5cc82f65c70b06d4e803ccf649193ef0.g.cs:282 
  at Windows.UI.Xaml.Controls.NavigationView.RaiseItemInvoked (System.Object item, System.Boolean isSettings, Windows.UI.Xaml.Controls.NavigationViewItemBase container, Windows.UI.Xaml.Controls.NavigationView+NavigationRecommendedTransitionDirection recommendedDirection) [0x00093] in D:\code\framework\uno\src\Uno.UI\UI\Xaml\Controls\NavigationView\NavigationView.cs:1428 
  at Windows.UI.Xaml.Controls.NavigationView.ChangeSelection (System.Object prevItem, System.Object nextItem) [0x000ac] in D:\code\framework\uno\src\Uno.UI\UI\Xaml\Controls\NavigationView\NavigationView.cs:1330 
  at Windows.UI.Xaml.Controls.NavigationView.OnSelectedItemPropertyChanged (Windows.UI.Xaml.DependencyPropertyChangedEventArgs args) [0x00008] in D:\code\framework\uno\src\Uno.UI\UI\Xaml\Controls\NavigationView\NavigationView.cs:1879 
  at Windows.UI.Xaml.Controls.NavigationView.OnPropertyChanged (Windows.UI.Xaml.DependencyPropertyChangedEventArgs args) [0x00082] in D:\code\framework\uno\src\Uno.UI\UI\Xaml\Controls\NavigationView\NavigationView.cs:2658 
  at Windows.UI.Xaml.Controls.NavigationView+<>c.<.cctor>b__458_27 (Windows.UI.Xaml.DependencyObject s, Windows.UI.Xaml.DependencyPropertyChangedEventArgs e) [0x00000] in D:\code\framework\uno\src\Uno.UI\UI\Xaml\Controls\NavigationView\NavigationView.Properties.cs:491 
  ...
```

**Comment by davidjohnoliver** [comment]
Can you create an issue on the Uno repo, since it's coming from an Uno bug?

**Comment by carldebilly** [comment]
I fixed this NRE in Uno.WCT last week.  It's not a Uno bug, it's by design how it's supposed to work.  Test on UWP, it's working the same way!

**Comment by XiaotianNetlift** [comment]
closed by #54
